test/cmdlineTests.sh

```diff
@@ -641,6 +674,25 @@ printTask "Testing the eqivalence of --via-ir and a two-stage compilation..."
     done
 )
 
+printTask "Testing the eqivalence of CLI options and Standard JSON settings..."
```

Comment:
nice! would you say that there are potentially more differences? would it make sense to add also other tests?

Reply:
I don't think there are that many differences - I can't think of any off the top of my head - but since this is not covered by tests, I bet we do have some.

As for more tests, I don't think it makes sense to spend time adding complete coverage here and I'd only add them for options being added or modified. There are just too many combinations.